lein-jank/bin/ci/test

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -euo pipefail
+lein install
+cd test-project
+lein jank run 1 | grep 'lein jank works!'
```

Comment:
Everything looks good. Just wanted to double check this, though. lein-jank is going to expect that `jank` is available on the `PATH` and I don't think it is. We'd need to export a changed `PATH` before running this, right?

Reply:
The invariant I originally assumed was that jank would be installed before calling this script, which is why I added the install step in the workflow. Are there other similar scripts with different conventions?

Reply:
Oh, right. You're installing globally, not to a `DESTDIR`. This may just work then.